### PR TITLE
docs: updates to whats new 7.1

### PR DESCRIPTION
**Pull request by daniellee**
Added some missing features, fixed links and rewrote some sentences. Also reordered the items to match the release blog post.

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 1]
```diff
@@ -16,51 +16,49 @@ This topic includes the release notes for the Grafana v7.1, which is currently i
...
 - [**Grafana Enterprise features**]({{< relref "#grafana-enterprise-features" >}})
-  - [**Support for HashiCorp Vault**]({{< relref "#support-for-hashicorp-vault" >}})
-  - [**Internal links for Elastic**]({{< relref "#internal-links-for-elastic" >}})
-
-## Query history search
+  - [**Secret management with HashiCorp Vault**]({{< relref "#support-for-hashicorp-vault" >}})
+  - [**Monthly schedules in reports**]({{< relref "#support-for-monthly-schedules-in-reports" >}})
 
-In Grafana v 7.1 we are introducing search functionality in Query history. You can search across queries and your comments. It is especially useful in combination with a time filter and data source filter. Read more about Query history [here]({{<relref "../features/explore/index.md#query-history" >}}).
+## Influx Datasource
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-1.md:33]:
```suggestion
## Influx data source
```

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 2]
```diff
@@ -16,51 +16,49 @@ This topic includes the release notes for the Grafana v7.1, which is currently i
...
+## Query history search
 
-Grafana v7.1 adds support for provisioning of app plugins. This allows app plugins to be configured and enabled/disabled using configuration files. Read more about provisioning of app plugins [here]({{<relref "../administration/provisioning.md#plugins" >}}).
+In Grafana v 7.1 we are introducing search functionality in Query history. You can search across queries and your comments. It is especially useful in combination with a time filter and data source filter. Read more about [Query history here]({{<relref "../features/explore/index.md#query-history" >}}).
 
-## Azure Monitor Datasource
+{{< docs-imagebox img="/img/docs/v71/query_history_search.gif" max-width="800px" caption="Query history search" >}}
 
-Support for multiple dimensions has been added to all services in the Azure Monitor datasource. This means you can now group by more than one dimension with time series queries. With the Kusto based services, Log Analytics and Application Insights Analytics, you can also select multiple metrics as well as multiple dimensions.
+## Explore modes unified
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-1.md:43]: Explore needs to be updated to reflect this change.

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 3]
```diff
@@ -75,18 +73,25 @@ You can use the Text mode option to control what text the panel renders. If the
...
 {{< docs-imagebox img="/img/docs/v71/stat-panel-text-modes.png" max-width="1025px" caption="Stat panel" >}}
 
-## Explore modes unified
+## Provisioning of apps
 
-Grafana 7.1 includes a major change to Explore: it removes the query mode selector.
+Grafana v7.1 adds support for provisioning of app plugins. This allows app plugins to be configured and enabled/disabled using configuration files. Read more about provisioning of app plugins [here]({{<relref "../administration/provisioning.md#plugins" >}}).
 
-Many data sources tell Grafana whether a response contains time series data or logs data. Using this information, Explore chooses which visualization to use for that data. This means that you don't need to switch back and forth between Logs and Metrics modes depending on the type of query that you want to make. 
+## Azure Monitor Datasource
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-1.md:80]:
```suggestion
## Azure Monitor data source
```

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 4]
```diff
@@ -75,18 +73,25 @@ You can use the Text mode option to control what text the panel renders. If the
...
+
+Support for multiple dimensions has been added to all services in the Azure Monitor datasource. This means you can now group by more than one dimension with time series queries. With the Kusto based services, Log Analytics and Application Insights Analytics, you can also select multiple metrics as well as multiple dimensions.
+
+Additionally, the “Raw Edit” mode for Application Insights Analytics has been replaced with a new service in the drop down for the datasource and is called “Insights Analytics”. The new query editor behaves in the same way as Log Analytics.
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-1.md:84]:
```suggestion
Additionally, the Raw Edit mode for Application Insights Analytics has been replaced with a new service in the drop down for the data source and is called Insights Analytics. The new query editor behaves in the same way as Log Analytics.
```

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 5]
```diff
@@ -75,18 +73,25 @@ You can use the Text mode option to control what text the panel renders. If the
...
+
+## Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) datasource
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-1.md:86]:
```suggestion
## Deep linking for Google Cloud Monitoring (formerly named Google Stackdriver) data source
```

File: docs/sources/guides/whats-new-in-v7-1.md [hunk 6]
```diff
@@ -96,7 +101,7 @@ General features are included in the Grafana Enterprise edition software.
 
 You can now use HashiCorp Vault to get secrets for configuration and provisioning of Grafana Enterprise. Learn more about this feature [here]({{<relref "../enterprise/vault.md">}}).
 
-### Support for monthly in reports
+### Support for monthly schedules in reports
 
 With Grafana Enterprise 7.1 you can configure reports to be generated on a [monthly schedule]({{<relref "../enterprise/reporting.md#scheduling">}}).
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v7-1.md:106]:
```suggestion
With Grafana Enterprise 7.1, you can generate reports on a [monthly schedule]({{<relref "../enterprise/reporting.md#scheduling">}}).
```

**Review by oddlittlebird — APPROVED** [review]
A few minor edits and at least one missing link/content update.

File: docs/sources/guides/whats-new-in-v7-1.md (hunk 2 above)
Comment (daniellee) [reply docs/sources/guides/whats-new-in-v7-1.md:43]: @oddlittlebird not sure what you mean?

Comment (aocenas) [reply docs/sources/guides/whats-new-in-v7-1.md:43]: Hmm yeah I don't think we update the general explore docs after this. @kaydelaney can you take a look what needs to be updated there?

**Review by aocenas — APPROVED** [review]

File: docs/sources/guides/whats-new-in-v7-1.md (hunk 2 above)
Comment (daniellee) [reply docs/sources/guides/whats-new-in-v7-1.md:43]: Oh, do you mean the docs for Explore are not updated yet?

Comment (oddlittlebird) [reply docs/sources/guides/whats-new-in-v7-1.md:43]: Yes, and then linked from this document. I thought I saw this doc edit go through. Maybe it hasn't been merged yet?

**Pull request merged**